tests/test.yml

```diff
@@ -2,5 +2,6 @@
 - hosts: localhost
   remote_user: root
   roles:
-    - role: ansible-role-virtualenvwrapper
-      virtualenvwrapper_shell_rc_file: /home/travis/.bashrc
\ No newline at end of file
+    # Current parameterized role, look at '../defaults/main.yml' to see all variables
+    - role: ../..
+      virtualenvwrapper_shell_rc_file: /home/travis/.bashrc
```

Comment:
Marcolino thank you for you contribution! How about adding a test when the parameters 'virtualenvwrapper_venvs_project' and  'virtualenvwrapper_venvs_python' are set?
![screen shot 2015-07-10 at 3 39 12 pm](https://cloud.githubusercontent.com/assets/2402650/8627560/e7d1f8a8-2719-11e5-8675-ca977e01dd1e.png)